app/views/manuals/index.html.erb

```diff
@@ -5,7 +5,15 @@
     title: presented_manual.full_title,
   )
 %>
-<% if presented_manual.beta? %>
+
+<% if show_tasklist_header? %>
+  <%= render "govuk_component/task_list_header", {
+    title: "Learn to drive a car: step by step",
+    path: "/learn-to-drive-a-car"
```

Comment:
We need to add the skip link details:
 `skip_link: "#check-you're-allowed-to-drive",
  skip_link_text: "Check you're allowed to drive"`

Reply:
This page is a full width page, so there isn't a sidebar tasklist, meaning we don't "skip_link" this one.